src/transformers/trainer_pt_utils.py

```diff
@@ -1210,6 +1210,11 @@ class AcceleratorConfig:
         },
     )
 
+    mixed_precision: str = field(
+        default="no",
+        metadata={"help": "Whether or not use mixed precision and what type. Generally already set by TrainerArguments or during `accelerate launch`"},
```

Comment:
This should specify: 
- What it means if "no" is set 
- What takes precendence from AcceleratorConfig, trainer arguments or accelerate launch i.e. what if it's set here to fp16 and set to "no" in the trainer arguments?